src/js/components/leftPanel/analysisPanel/BaseAnalysis.tsx

```diff
@@ -82,20 +87,76 @@ const BaseAnalysis = (): JSX.Element => {
     }
   }
 
-  const AnalysisOptions = (): JSX.Element => (
-    <select
-      value={selectedAnalysis}
-      onChange={e => setSelectedAnalysis(e.target.value)}
-    >
-      {analysisModules.map((module: any, i: number) => {
-        return (
-          <option value={module.analysisId} key={i}>
-            {module.label.en}
-          </option>
-        );
-      })}
-    </select>
-  );
+  const AnalysisInstructions = (): JSX.Element | null => {
+    const currentAnalysis = analysisModules.find(
+      module => module.analysisId === selectedAnalysis
+    );
+    console.log(currentAnalysis);
```

Comment:
@vaidotasp there are a few `console.log()`, so I was thinking we could pair them down? If it's more beneficial to keep them, that's totally fine too. I've kept a few in previous PRs and found it to be very helpful